coordinator/src/cli.rs

```diff
@@ -17,6 +20,9 @@ pub struct Opts {
     /// Where to permanently store data, defaults to the current working directory.
     #[clap(long)]
     data_dir: Option<PathBuf>,
+
+    #[clap(long)]
```

Comment:
If you add a doc comment here it will be printed when providing an invalid argument/input  :)